guides/v2.2/cloud/env/variables-global.md

```diff
@@ -29,6 +29,9 @@ stage:
     MIN_LOGGING_LEVEL: debug
 ```
 
+{: .bs-callout .bs-callout-warning }
+Doesn't effect log level for the file handler which is debug by default.
```

Comment:
The `MIN_LOGGING_LEVEL` setting does not change the log level configuration for the file handler, which is set to `debug` by default.